std/protocols/permutation_via_bus.asm

```diff
@@ -18,7 +18,7 @@ let permutation_send: expr, Constr -> () = constr |id, permutation_constraint| {
 let permutation_receive: expr, Constr -> () = constr |id, permutation_constraint| {
     let (lhs_selector, lhs, rhs_selector, rhs) = unpack_permutation_constraint(permutation_constraint);
     
-    bus_receive(id, rhs, rhs_selector);
+    bus_receive_with_latch(id, rhs, rhs_selector, rhs_selector);
```

Comment:
@georgwiese I wasn't sure of this one

Reply:
Hmm, yeah, it's a bit annoying, because in a native permutation we only have a selector, and it is typically set to `LATCH * sel`, which is an expression that can only be nonzero in the latch row. An alternative solution is to set the selector to `sel` and add a constraint `(1 - LATCH) * sel = 0`. In that case, the latch doesn't appear in the permutation constraint at all, and witgen would currently fail.

I think we should start being explicit about the latch. From that perspective, it would make sense for this function to receive it explicitly, in addition to the permutation constraint.

Reply:
`@Schaeff .wait?;`